File: examples-sdk/zig/701-mq-rabbitmq-pubsub/main.zig
// 701-mq-rabbitmq-pubsub — Zig SDK equivalent
// Compile: vil compile --from zig --input 701-mq-rabbitmq-pubsub/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}

File: examples-sdk/go/701-mq-rabbitmq-pubsub/main.go
// 701-mq-rabbitmq-pubsub — Go SDK equivalent
// Compile: vil compile --from go --input 701-mq-rabbitmq-pubsub/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("app", 8080)
	s.Compile()
}
